README.md

```diff
@@ -17,7 +17,7 @@ For the latest news and updates, follow [@FacesBridgeSpec](https://twitter.com/F
 
 Using [Maven](https://maven.apache.org/) 3.x:
 
-	mvn clean install
+    mvn clean install
```

Comment:
@stiemannkj1 Is there a reason why spaces are better than tabs? The Markdown documentation says that 4 spaces or 1 tab will work: https://daringfireball.net/projects/markdown/syntax#precode